File: Move all negative elements to end/sol.java
class Solution {
    
    public void segregateElements(int arr[], int n)
    {
        ArrayList<Integer> ans = new ArrayList<>();
        for(int i=0; i<n; i++){
            if(arr[i] > 0){
                ans.add(arr[i]);
            }
        }
        
        for(int j=0; j<n; j++){
            if(arr[j] < 0){
                ans.add(arr[j]);
            }
        }
        
        for(int i=0; i<n; i++){
            arr[i] = ans.get(i);
        }
    }
}

File: Move all negative elements to end/sol.py
class Solution:
    def segregateElements(self, arr, n):
        ans = []
        
        # Add positive elements to the list
        for i in range(n):
            if arr[i] > 0:
                ans.append(arr[i])
        
        # Add negative elements to the list
        for j in range(n):
            if arr[j] < 0:
                ans.append(arr[j])
        
        # Update the original array with the elements from the list
        for i in range(n):
            arr[i] = ans[i]
